### ISSUE TITLE
I can't get the results in the paper using the pretrained models you provided

### ISSUE THREAD

**Issue by XY-ATOE** [Hzzone/ProPos#6]
I got the ACC 0.92 on cifar10 and 0.57 on cifar20 using the pre-trained model, while the paper had 0.94 and 0.61.
I use the features after projection head from the target encoder (ema-updated one). My transforms are resize(32) and normalize.The kmeans settings are the same as yours.

Finally, could you provide the training config on the stl10, imagenet10, imagenet-dogs, and tiny-imagenet?
Thank you

**Comment by Hzzone** [comment]
Hi, the results you mentioned are obtained with Resnet-34 in Table 2, while the pre-trained models are Resnet-18 in Table 6. In addition, plz make sure that the full datasets are used for evaluation.

**Comment by XY-ATOE** [comment]
> Sorry, I missed >_<. So, could you provide the configs or pretrain models of other datasets?

**Comment by Hzzone** [comment]
You can change the config on CIFAR-20, and the data augmentation.

**Comment by XY-ATOE** [comment]
I used resnet18 to train imagenet-dogs and I got acc is 0.63. could I directly replace resnet18 with resnet34 if I need to train resnet34 on the dataset. The acc I get from doing this is only 0.58.

**Comment by Hzzone** [comment]
Could you post the configs you have used? I will update the configs with R34 if time is available.

**Comment by XY-ATOE** [comment]
I only changed backbone and image size basised on CIFAR20_R18 config.
```python
batch_size: 64
num_devices: 2
momentum_base: 0.996
momentum_max: 1.0
momentum_increase: true
dataset: imagenet-dog
eval_metric:
  - nmi
  - acc
  - ari
whole_dataset: true
encoder_name: bigresnet34
epochs: 1000
feat_dim: 256
hidden_size: 4096
img_size: 96
lambda_predictor_lr: 10
learning_rate: 0.05
learning_eta_min: 0.
reassign: 1
save_freq: 50
save_checkpoints: true
shuffling_bn: true
symmetric: true
temperature: 0.5
use_gaussian_blur: false
warmup_epochs: 50
weight_decay: 0.0005
dist: true
data_resample: true
v2: true

model_name: propos
cluster_loss_weight: 0.1
latent_std: 0.001

wandb: false
project_name: 'clustering_archives'
entity: 'zzhuang'

```

**Comment by Hzzone** [comment]
Update the pre-trained models and configs on imagenetdogs.
